README.md

```diff
@@ -20,17 +20,42 @@ QGreenland Team to data and user communities.
 
 ## Disclaimer
 
-QGreenland should not be used as a sole navigational aid while performing remote
-research activities in Greenland. Always bring appropriate saftey and
+QGreenland should not be used as a sole navigational aid while performing
+remote research activities in Greenland. Always bring appropriate safety and
 navigational aids (including hard-copies of topographic maps) when visiting the
 field.
 
-QGreenland's layers may contain errors from the original data providers.
-QGreenland makes no guarantees about the accuracy and validity of data
-contained in QGreenland. For example, some layers may not perfectly align with
-each other due to unidentified georeferencing issues. We recommend using the
-'Greenland Coastlines' layer as the reference layer for geolocating Greenland's
-coastline.
+QGreenland is a data-viewing platform, and the QGreenland Team does not create
```

Comment:
I think we should change this to `QGreenland is a data-viewing and analysis platform`.

Reply:
9c34bea9